File: eBarberShop/eBarberShop.Model/Slike.cs
namespace eBarberShop.Model
{
    public class Slike
    {
        public int SlikeId { get; set; }
        public string? Opis { get; set; }
        public byte[] Slika { get; set; }
        public DateTime DatumPostavljanja { get; set; }

        //public virtual ICollection<SlikeUsluge> SlikeUsluge { get; set; } = new List<SlikeUsluge>();
    }
}

File: eBarberShop/UI/ebarbershop_desktop/lib/models/slike/slike.dart
import 'package:json_annotation/json_annotation.dart';

part 'slike.g.dart';

@JsonSerializable()
class Slike {
  int slikeId;
  String? opis;
  String slika;
  DateTime datumPostavljanja;

  Slike(this.slikeId, this.opis, this.slika, this.datumPostavljanja);

  factory Slike.fromJson(Map<String, dynamic> json) =>
      _$SlikeFromJson(json);

  Map<String, dynamic> toJson() => _$SlikeToJson(this);
}
